src/catalogue/items/catalogueItemsDialog.component.tsx

```diff
@@ -804,82 +780,70 @@ function CatalogueItemsDialog(props: CatalogueItemsDialogProps) {
-                    setFormError(false);
-                    setFormErrorMessage(undefined);
-                    setManufacturerAddressError(false);
-                  }}
-                  error={manufacturerAddressError} // Set error state based on the nameError state
-                  helperText={
-                    manufacturerAddressError
-                      ? 'Please enter a Manufacturer Address'
-                      : ''
-                  }
-                  fullWidth
-                />
+
+              <Grid item xs={12} style={{ display: 'flex' }}>
+                <Grid item xs={11}>
+                  <Autocomplete
+                    value={
+                      //logic means that current manufacturer renders in edit dialog, but behaves the same as add dialog (so can be changed/cleared)
+                      selectedCatalogueItemManufacturer &&
+                      selectedManufacturer === null &&
+                      inputValue !== ''
+                        ? selectedCatalogueItemManufacturer
+                        : selectedManufacturer
+                    }
+                    inputValue={inputValue ?? ''}
+                    onInputChange={(event, newInputValue) =>
+                      setInputValue(newInputValue)
+                    }
+                    onChange={(
+                      event: any,
+                      newManufacturer: Manufacturer | null
+                    ) => {
+                      setSelectedManufacturer(newManufacturer ?? null);
+                      setInputValue(newManufacturer?.name ?? '');
+                      handleCatalogueDetails(
+                        'manufacturer_id',
+                        newManufacturer?.id ?? null
+                      );
+                    }}
+                    id="manufacturer-autocomplete"
+                    options={manufacturerList ?? []}
+                    size="small"
+                    isOptionEqualToValue={(option, value) =>
+                      option.name === value.name
+                    }
+                    getOptionLabel={(option) => option.name}
+                    renderInput={(params) => (
+                      <TextField
+                        {...params}
+                        required={true}
+                        label="Manufacturer"
+                        error={errorMessages.manufacturer_id !== undefined}
+                        helperText={
+                          errorMessages.manufacturer_id !== undefined
+                            ? errorMessages.manufacturer_id
+                            : ''
+                        }
+                      />
+                    )}
+                  />
+                </Grid>
+                <Grid item xs={1}>
+                  <IconButton
+                    sx={{ mx: '4px', my: '2px' }}
+                    onClick={() => setAddManufacturerDialogOpen(true)}
+                    disabled={selectedManufacturer !== null}
```

Comment:
Why is this button being disabled?
It is not consistent as it disables the button when you selected a manufacturer when your adding a catalogue item but with save as and edit the button is enabled. look at gif below 

  
![man ufacturer add in cat ](https://github.com/ral-facilities/inventory-management-system/assets/83226114/cc229ad7-024a-4415-a081-7c9f9b940dfd)